src/delete/mod.rs

```diff
@@ -16,14 +16,7 @@ impl Delete for ImageKit {
     async fn delete<T: ToString + Send>(&self, file_id: T) -> Result<()> {
         let url_string = format!("{}/{}", FILES_ENDPOINT, file_id.to_string());
         let endpoint_url = Url::parse(&url_string).unwrap();
-        let private_key = self.private_key.to_owned();
-        let response = self
-            .client
-            .delete(endpoint_url)
-            .basic_auth::<String, String>(private_key, None)
-            .send()
-            .await
-            .unwrap();
+        let response = self.client.delete(endpoint_url).send().await.unwrap();
```

Comment:
Would be better if we handle the result with error propagation instead of unwrapping. It's not completely infallible and the error should be handled by the client.

`let response = self.client.delete(endpoint_url).send().await?;`

Reply:
Gotcha! You are right! I went over this while working to embrace testing! Let me check quick!

Reply:
Solved!  13de9a7